app/src/main/java/com/example/mykotlincalculator21/mvp/view/CalculatorView.kt

```diff
@@ -0,0 +1,18 @@
+package com.example.mykotlincalculator21.mvp.view
+
+import com.example.mykotlincalculator21.MainActivity
+import com.example.mykotlincalculator21.databinding.ActivityMainBinding
+import com.example.mykotlincalculator21.mvp.contract.CalculatorContract
+
+class CalculatorView(mainActivity: MainActivity, binding: ActivityMainBinding): ActivityView(mainActivity), CalculatorContract.CalculatorViewContract {
```

Comment:
same here.. you can add private val in the constructor and delete the init statement and the variable

Reply:
Done. Thx